onnx/backend/test/case/node/__init__.py

```diff
@@ -22,22 +22,28 @@
 from onnx.onnx_operators_pb import FunctionProto
 
 
+# FIXME(TMVector): Any reason we can't get rid of this and use the C++ helper directly?
```

Comment:
TODO/question

Reply:
The changes look good to me, thanks! I guess we can import the C++ version into Python (provided there is no subtle difference between them due to bugs, etc., I suppose). I guess that could be a different PR, given that we are close to the cutoff date for the next ONNX release.